recipe/meta.yaml

```diff
@@ -22,7 +24,7 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - setuptools
     - wheel
```

Comment:
in the run section imagecodecs should be `imagecodecs>=2021.7.30 `, see https://github.com/cgohlke/tifffile/blob/0eadc15ca044429f8af70a59ae9cc5a3d54b18e2/setup.py#L89